File: Simple Object Oriented Language Examples/class-example.py
#!/usr/bin/env python

'''

Author: wetw0rk

Very simple example of object oriented programming in Python

'''

class Picture():

  def __init__(self, id, title, img):
    self.id     = id
    self.title  = title
    self.img    = img

  def display_values(self):
    print("id:%s, title:%s, img:%s" % (
          self.id, self.title, self.img
      )
    )

# initialize and call
example = Picture("1", "wetw0rk", "file.png")
example.display_values()

File: Simple Object Oriented Language Examples/class-example.php
<?php

/*

Author: wetw0rk

Very simple example of object oriented programming in PHP

*/

class Picture
{
  function __construct($id, $title, $img)
  {
    $this->id     = $id;
    $this->title  = $title;
    $this->img    = $img;
  }
  function display_values()
  {
    echo sprintf("id:%s, ", $this->id);
    echo sprintf("title:%s, ", $this->title);
    echo sprintf("img:%s\n", $this->img);
  }
}

// initialize and call
$example = new Picture("1", "wetw0rk", "file.png");
$example->display_values();

?>
